Tools/TestWebKitAPI/Tests/WebKitCocoa/SOAuthorizationTests.mm

```diff
@@ -54,6 +54,7 @@
 static bool uiShowed = false;
 static bool newWindowCreated = false;
 static bool haveHttpBody = false;
+static RetainPtr<NSData> httpBody;
```

Comment:
This should be an ivar of the delegate

Reply:
Not really, the SOA plugin is using static functions